sim-lib/src/cln.rs

```diff
@@ -1,18 +1,24 @@
+use std::time::Duration;
```

Comment:
You can import `Duration` from `tokio::time` given we are already adding that dependency.

Reply:
good point